--- sendmail-8.18.1/libsharedmilter/README	2024-12-04 10:03:13.495177920 +0200
+++ sendmail-8.18.1/libsharedmilter/README.new	2024-12-04 10:58:33.246231210 +0200
@@ -12,6 +12,9 @@
 Starting with 8.13 sendmail is compiled by default with support for
 the milter API.
 
+Starting with 8.18.2 a libmilter shared object can be built
+(`libmilter.so'). A custom version number can be used (see below).
+
 Note: if you want to write a milter in Java, then see
 http://sendmail-jilter.sourceforge.net/
 
@@ -38,7 +41,6 @@
 SM_CONF_POLL
 	Set to 1 if poll(2) should be used instead of select(2).
 
-
 +-------------------+
 | BUILDING A FILTER |
 +-------------------+
@@ -67,6 +69,11 @@
 setrlimit() to increase the number of open file descriptors if your filter
 is going to be busy.
 
+A custom version number 'n' can be defined for `libmilter.so.n'. It should
+consist of only numbers and dots (e.g. "4", or "2.0.1").
+
+To use a custom SO version number, define `confCUSTOM_MILTER_SOVER'
+in `site.config.m4'.
 
 +----------------------------------------+
 | SPECIFYING FILTERS IN SENDMAIL CONFIGS |
