--- devtools/README	2024-12-08 14:38:45.083144209 +0200
+++ devtools/README.new	2024-12-08 14:38:34.532857293 +0200
@@ -51,6 +51,17 @@
 and their defaults before referencing the appropriate OS definitions
 are listed below.
 
+conf_libmilter_CUSTOM_SOMAJOR	[empty]	Define a custom (MAJOR) version
+					number N for a shared object
+					(libmilter.so.N).
+conf_libmilter_CUSTOM_SOMINOR	[empty]	Define a custom MINOR version number
+					N for a shared object (libmilter.x.N).
+					Requires a custom MAJOR version number. 
+conf_libmilter_CUSTOM_SOPATCH	[emtpy]	Define a custom PATCHLEVEL version
+					number N for a shared object
+					(libmilter.x.x.N).
+					Requires a custom MAJOR and MINOR
+					version number.
 confBEFORE	    [empty]		Files to create before sendmail is
 					compiled.  The methods must be defined
 					in the Makefile using PUSHDIVERT(3).
